tools/codegen/api/cpp.py

```diff
@@ -146,7 +150,7 @@ def returns_type(rs: Sequence[Return]) -> CType:
     else:
         return TupleCType([return_type(r) for r in rs])
 
-def return_names(f: NativeFunction) -> Sequence[str]:
+def return_names(f: NativeFunction, *, override_name: Optional[str] = None) -> Sequence[str]:
```

Comment:
override_name doesn't seem all that appropriate here as half the time you don't actually override when this is non-None! Maybe `fallback_name="result"` (and then you can also dump "result") below.